tools/x-docs/src/docs/guides/components/cookbook/styling.md

```diff
@@ -1,7 +1,94 @@
 # Styling
 
-Currently, there's no officially-supported way of providing styling for components. To begin with, all our components that require styling are built to be compatible with Origami styles, and the recommended way of using these is by including the CSS from the [Origami Build Service](https://www.ft.com/__origami/service/build/v2/) and referencing the classes in your markup.
+## Origami components
 
-For styles specific to a single x-dash component, we suggest writing a CSS file and including it in the package, then importing it into your app however you normally include styles (e.g. using a SCSS `@import`).
+Some x-dash components, such as [x-teaser](/components/x-teaser/readme.md), are built to be compatible with styles provided by Origami components, by using their classnames. These styles ase expected to be provided by the application that's consuming the x-dash component, for example by including the CSS from the [Origami Build Service](https://www.ft.com/__origami/service/build/v2/). To include these styles in a component's demos in the documentation and component explorer, use the `dependencies` object in your component's [`stories/index.js`](/tools/x-docs/guides/components/setup.md#indexjs).
```

Comment:
"ase" => are

"in your component's story configuration"?

Reply:
i wanted to make it clear it goes in `index.js` not the individual stories?